src/System.Private.ServiceModel/tools/test/BridgeCertificateBootstrapper/Bootstrapper.cs

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Infrastructure.Common;
+
+public static class BridgeCertificateBootstrapper
+{
+    public static int Main(string[] args)
+    {
+        DisplayBanner(); 
+
+        if (args.Length < 1)
+        {
+            DisplayUsage();
+            return -1;
+        }
+
+        var filename = args[0];
+
+        byte[] certificateBytes; 
+        if (GetCertificate(out certificateBytes))
+        {
+            File.WriteAllBytes(filename, certificateBytes);
+            return 0;
+        }
+
+        Console.WriteLine();
+        return -1; 
+    }
+
+    public static bool GetCertificate(out byte[] certificateBytes)
+    {
+        string endpoint = Endpoints.Https_DefaultBinding_Address;
+        string bridgePort = TestProperties.GetProperty(TestProperties.BridgePort_PropertyName);
+        
+        certificateBytes = default(byte[]);
+
+        Uri uri;
+        try
+        {
+            if (Uri.TryCreate(endpoint, UriKind.RelativeOrAbsolute, out uri))
+            {
+                string certAsPemUri = string.Format(
+                    "http://{0}:{1}/resource/WcfService.CertificateResources.CertificateAuthorityResource?exportAsPem=true", uri.Host, bridgePort);
+
+                Console.WriteLine("  Bridge hostname is: '{0}'", uri.Host);
+                Console.WriteLine("  Retrieving certificate from:");
+                
+                Console.WriteLine(certAsPemUri);
+                Console.WriteLine();
+
+                HttpClient client = new HttpClient();
+                var response = client.GetAsync(certAsPemUri).GetAwaiter().GetResult();
+
+                certificateBytes = response.Content.ReadAsByteArrayAsync().GetAwaiter().GetResult();
+
+                Console.WriteLine("   ... success");
+                return true; 
```

Comment:
Nit: possibly say "successfully read {0} bytes from Bridge".  Then any error happening later won't have "success..." written to the console.  Optional.